sdk/eventhub/azure-eventhub/azure/eventhub/aio/_connection_manager_async.py

```diff
@@ -63,7 +63,7 @@ def __init__(
         **kwargs: Any,
     ) -> None:
         self._loop = kwargs.get("loop")
-        self._lock = Lock(loop=self._loop)
+        self._lock = Lock(loop=self._loop) # pylint:disable=unexpected-keyword-arg
```

Comment:
if this is not a public api can we just remove the loop being passed in ? Its been deprecated since 3.8 and removed from 3.10

Reply:
yup!